### DISCUSSION TITLE
Custom gruvbox slightly broken in edit mode with new live preview

### DISCUSSION THREAD

**Discussion by ryanbbrown** [BryanJenksCommunity/FAQ#136] [Q&A] [answered]
I'm using your gruvbox theme, but since the update that gives obsidian native live preview functionality, the preview font is the default Obsidian font instead of the gruvbox font; it only shows the gruvbox font in actual preview mode. Better bullet points also no longer show during edit mode. Is there a way to fix this other than using the legacy editor?

**Edit mode:**
![image](https://user-images.githubusercontent.com/56168848/147370254-c65c4dbb-7eb5-4d12-a74b-7a92bed660a3.png)

**Preview mode:**
![image](https://user-images.githubusercontent.com/56168848/147370260-25815770-a92b-46b8-98b5-febb92477259.png)

**Comment by tallguyjenks** [comment, marked as answer]
this is a difference in how the CSS selectors work between the different modes, not sure when I'll get around to fixing it but it shouldn't be a terribly problematic fix to make the selectors for live preview mode also use the same content as the original preview mode
